packages/Slider/Range.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { ChangeEvent, useCallback, useEffect, useRef, useState } from 'react'
+import { Box } from '@welcome-ui/box'
+import { Text } from '@welcome-ui/text'
+import { Hint } from '@welcome-ui/hint'
+import { InputText } from '@welcome-ui/input-text'
+import { CreateWuiProps, forwardRef } from '@welcome-ui/system'
+
+import * as S from './styles'
+
+import { SliderOptions } from './index'
+
+export type Range = {
+  min: number
+  max: number
+}
+
+export interface RangeOptions extends Omit<SliderOptions, 'type' | 'value' | 'onChange'> {
+  value: Range
+  onChange: (value: Range) => void
+  type?: 'inline' | 'fields'
+}
+
+export type RangeProps = CreateWuiProps<'div', RangeOptions>
+
+const round = (value: number, step = 1): number => Math.round(value / step) * step
+
+/**
+ * @name Slider.Range
+ */
+export const Range = forwardRef<'div', RangeProps>(
+  ({
+    borderSelectorColor = 'light.900',
+    disabled,
+    hint,
+    label,
+    max,
+    min,
+    onChange,
+    step = 1,
+    tooltip,
+    type,
+    value,
+    values,
+    ...restProps
+  }) => {
+    const [minValue, setMinValue] = useState(min)
+    const [maxValue, setMaxValue] = useState(max)
+    const minValueRef = useRef<HTMLInputElement>(null)
+    const maxValueRef = useRef<HTMLInputElement>(null)
+    const range = useRef<HTMLDivElement>(null)
+    const tooltipMinRef = useRef<HTMLOutputElement>(null)
+    const tooltipMaxRef = useRef<HTMLOutputElement>(null)
+    const [tooltipMinVisible, setTooltipMinVisible] = useState<boolean>(false)
+    const [tooltipMaxVisible, setTooltipMaxVisible] = useState<boolean>(false)
+
+    const handleMinValue = (e: ChangeEvent<HTMLInputElement>) => {
+      // Prevents the min value from being above the max
+      const value = round(Math.min(parseInt(e.target.value, 10), maxValue - 1), step)
```

Comment:
No need to verify if `isNan(e.target.value)`? (Same question for handleMaxValue)

Reply:
As the min & max are forced to be `number`, I don't think we have to check if the `parseInt` result to a `NaN`.
We need to do this conversion because of the `input` doing the inverse thing (use `number` attributes & convert them to `string` for event `value`)